packages/app-project/src/shared/components/SubjectPreview/SubjectPreview.js

```diff
@@ -5,84 +5,87 @@ import { FavouritesButton, Media } from '@zooniverse/react-components'
 import { CollectionsButton, TalkLink } from './components'
 
 const defaultSubject = {
-  favorite: false,
-  id: '',
-  toggleFavorite: () => false,
-  locations: []
+	favorite: false,
+	id: '',
+	toggleFavorite: () => false,
+	locations: []
 }
-function SubjectPreview ({
-  height = '250px',
-  isLoggedIn = false,
-  placeholder,
-  slug,
-  subject = defaultSubject,
-  width = '400px'
+function SubjectPreview({
+	height = '250px',
+	isLoggedIn = false,
+	placeholder,
+	slug,
+	subject = defaultSubject,
+	width = '400px'
 }) {
-  const subjectURLs = subject.locations.map(location => Object.values(location)[0])
-  const subjectURL = subjectURLs[0]
-  const collectionsModal = createRef()
-  const href = `/projects/${slug}/talk/subjects/${subject.id}`
+	const subjectURLs = subject.locations.map(location => Object.values(location)[0])
+	const subjectURL = subjectURLs[0]
+	const collectionsModal = createRef()
+	const href = `/projects/${slug}/talk/subjects/${subject.id}`
 
-  return (
-    <Box
-      fill
-    >
-      <Anchor
-        href={href}
-      >
-        <Box
-          align='center'
-          height={height}
-          overflow='hidden'
-          width={width}
-        >
-          <Media
-            alt={`subject ${subject.id}`}
-            height={700}
-            placeholder={placeholder}
-            src={subjectURL}
-            width={700}
-          />
-        </Box>
-      </Anchor>
-      <Box direction='column' gap='xsmall' pad={{ vertical: 'xsmall' }}>
-        <TalkLink
-          href={href}
-        />
-        <FavouritesButton
-          checked={subject.favorite}
-          disabled={!isLoggedIn}
-          onClick={subject.toggleFavourite}
-        />
-        <CollectionsButton
-          disabled={!isLoggedIn}
-          subject={subject}
-        />
-      </Box>
-    </Box>
-  )
+	return (
+		<Box
+			fill
+		>
+			<Anchor
+				role="subject-preview-link"
+				href={href}
+			>
+				<Box
+					align='center'
+					height={height}
+					overflow='hidden'
+					width={width}
+					role="shit"
```

Comment:
Not sure what this is doing.

Reply:
De-escalating frustration... lol. Removed!